src/ansys/dpf/core/operators/result/magnetic_vector_potential.py

```diff
@@ -1,5 +1,5 @@
 """
-acceleration_Z
```

Comment:
Why is the `acceleration_z` operator considered as renamed to `magnetic_vector_potential.py`?